git/package.sls

```diff
@@ -3,3 +3,4 @@
 git:
   pkg.installed:
     - name: {{ git.get('git', 'git') }}
+    - reload_modules: True
```

Comment:
This is not needed. Modules only need to be reloaded if salt doesn't know that a system wide package was installed (example by using cmd).